test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -498,6 +500,22 @@ public void deletePVC(String namespace, String name) {
         return exceptionallyExpecting(callback.getFuture(), isConflict, null);
     }
 
+    /**
+     * Create ConfigMap.
+     * @param namespace The namespace where the ConfigMap should be created.
+     * @param binding The cluster ConfigMap.
+     * @return A future indicating the status of the ConfigMap operation.
+     */
+    @SneakyThrows(ApiException.class)
+    public CompletableFuture<V1ConfigMap> createConfigMap(String namespace, V1ConfigMap binding) {
+        CoreV1Api api = new CoreV1Api();
+        //V1ConfigMap api = new V1ConfigMap ();
```

Comment:
Remove all commented out code. Here and everywhere else you used it.

Reply:
Removed.